src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -132,6 +133,25 @@ class IOUAmountPage extends React.Component {
         });
     }
 
+    /**
+     * Return length of Amount string without commas.
+     *
+     * @param {String} amount
+     * @returns {Number}
+     */
+    amountLength(amount) {
+        const trailingZeroes = amount.match(/^0+/);
```

Comment:
Should this be `leadingZeroes` ? if so, please replace all the trailing to leading.

Reply:
Yeah my bad!